Tests/TestingApps/PurchaseTesterSwiftUI/Shared/ReceiptVerifier.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ReceiptVerifier.swift
+//  ReceiptParserApp
+//
+//  Created by Andrés Boedo on 1/26/23.
+//
+
+import Foundation
+
+struct ReceiptVerifier {
+
+    private static let errorMessagesByCode = [
```

Comment:
not gonna win any awards for this file's code, but it does the job and it's a testing app and not SDK code

Reply:
Yeah this doesn't have to be super polished.